variables.tf

```
@@ -112,6 +112,12 @@ variable "apply_immediately" {
   description = "Apply changes immediately"
 }
 
+variable "data_tiering_enabled" {
+  type        = bool
+  default     = false
+  description = "Enables data tiering. Data tiering is only supported for replication groups using the r6gd node type. See: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/elasticache_replication_group#data_tiering_enabled"
```

You already copied this information from the link so I do not think we need it anymore.

```suggestion
  description = "Enables data tiering. Data tiering is only supported for replication groups using the r6gd node type."
```

---

After this change, you'll have to run `make pr/auto-format` again